### PR TITLE
Improve the SCIM2 Multi-attribute Fitlering Test-case

### PR DESCRIPTION
**Pull request by sarubi**
improvement of 
Fix https://github.com/wso2/product-is/pull/3601
fix https://github.com/wso2/product-is/pull/4441

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/SCIM2MultiAttributeUserFilterTestCase.java [hunk 1]
```diff
@@ -171,8 +173,7 @@ private void createUser(String familyName, String givenName, String userName, St
         request.setEntity(entity);
 
         HttpResponse response = client.execute(request);
-        assertEquals(response.getStatusLine().getStatusCode(), 201, "User " +
-                "has not been created successfully");
+        assertEquals(response.getStatusLine().getStatusCode(), 201, "User " + "has not been created successfully");
```
Comment (pamodaaw) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/SCIM2MultiAttributeUserFilterTestCase.java]: Shall we have "User" and the rest of the sentence as a single string

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/SCIM2MultiAttributeUserFilterTestCase.java [hunk 2]
```diff
@@ -218,8 +219,7 @@ private void createGroup(List<String> userNames, List<String> userIDs, String gr
 
         HttpResponse response = client.execute(request);
 
-        assertEquals(response.getStatusLine().getStatusCode(), 201, "User " +
-                "has not been created successfully");
+        assertEquals(response.getStatusLine().getStatusCode(), 201, "User " + "has not been created successfully");
```
Comment (pamodaaw) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/SCIM2MultiAttributeUserFilterTestCase.java]: Here as well

Comment (sarubi) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/SCIM2MultiAttributeUserFilterTestCase.java]: Fixed

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/SCIM2MultiAttributeUserFilterTestCase.java (hunk 1 above)
Comment (sarubi) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/SCIM2MultiAttributeUserFilterTestCase.java]: Fixed

**Pull request merged**